### PR TITLE
session name for bookmarks

### PR DESCRIPTION
**Pull request by pondrejk**
### Problem Statement
name.lower wasn't always equal with session name,  also removed some dead code and unused params

### Solution
making the session name explicit in constants

### CODE REVIEW

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 5920
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/ui/test_bookmarks.py -k test_positive_create_bookmark_public --external-logging
Test Result : = 8 failed, 30 passed, 156 deselected, 102 warnings, 1 error in 3791.45s (1:03:11) =
```

**Comment by pondrejk** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_bookmarks.py -k test_positive_create_bookmark_public

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 5922
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/ui/test_bookmarks.py -k test_positive_create_bookmark_public --external-logging
Test Result : = 6 failed, 30 passed, 1 skipped, 156 deselected, 99 warnings, 2 errors in 4429.18s (1:13:49) =
```

File: robottelo/constants/__init__.py [hunk 1]
```diff
@@ -1651,81 +1651,121 @@ class Colored(Box):
...
+    {'name': 'SyncPlan', 'controller': 'katello_sync_plans', 'session_name': 'syncplan'},
+    {'name': 'ContentView', 'controller': 'katello_content_views', 'session_name': 'contentview'},
+    {'name': 'Errata', 'controller': 'katello_errata', 'session_name': 'errata'},
+    {'name': 'Package', 'controller': 'katello_erratum_packages', 'session_name': 'package'},
+    {
+        'name': 'ContainerImageTag',
+        'controller': 'katello_docker_tags',
+        'session_name': 'containerimagetag',
+    },
+    {'name': 'Host', 'controller': 'hosts', 'setup': entities.Host, 'session_name': 'host_new'},
```
Comment (lhellebr) [inline robottelo/constants/__init__.py:1686]: Here's the example where session_name != name.lower()

**Review by lhellebr — APPROVED** [review]

**Comment by pondrejk** [comment]
the remaining failures are not due to the original problem, some more work on the airgun side is needed for those

**Review by vsedmik — APPROVED** [review]
Looks good to me, just needs rebase.
Also the CI/CQ has been fixed if I am not quite mistaken.

**Pull request merged**